src/raftstore/store/store.rs

```diff
@@ -287,6 +293,18 @@ impl<T, C> Store<T, C> {
         Ok(())
     }
 
+    fn clean_up_meta(&mut self, wb: &mut WriteBatch, region: &metapb::Region) {
```

Comment:
Why does `clean_up_meta` confuse you? What do you think `clean_up_meta` do? Actually the tombstone key exists already, but `peer_storage::clear_meta` will delete it, hence write it back again.